continuous_integration/hdfs/Dockerfile

```diff
@@ -7,11 +7,11 @@ RUN apt-get update && \
 # Install CDH5 in a single node: Pseudo Distributed
 # Docs: http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cdh_qs_yarn_pseudo.html
 ADD docker-files/cloudera.pref /etc/apt/preferences.d/cloudera.pref
-RUN curl -s http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh/archive.key | apt-key add - && \
-    echo 'deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list && \
-    echo 'deb-src http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh5 contrib' >> /etc/apt/sources.list.d/cloudera.list && \
+RUN curl -s http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh/archive.key | apt-key add - && \
+    echo 'deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh xenial-cdh5 contrib' > /etc/apt/sources.list.d/cloudera.list && \
+    echo 'deb-src http://archive.cloudera.com/cdh5/ubuntu/xenial/amd64/cdh xenial-cdh5 contrib' >> /etc/apt/sources.list.d/cloudera.list && \
     apt-get update && \
-    apt-get install -y -q openjdk-7-jre-headless hadoop-conf-pseudo && \
+    apt-get install -y -q sudo openjdk-8-jre-headless hadoop-conf-pseudo libhdfs0 && \
```

Comment:
Why is sudo listed here?

Reply:
The new ubuntu docker base image doesn't provide sudo by default anymore, while it is needed (afaict) to setup hdfs on docker.